documentation/manual/source/_theme.indigo/layout.html

```diff
@@ -32,25 +32,20 @@
     <h1><a href="http://qooxdoo.org/">qooxdoo</a></h1>
     <nav>
       <ul>
-        <!--
         <li>
-          <a class="" href="http://qooxdoo.org/">home</a>
+          <a class="" href="/demos.html">demos</a>
         </li>
-        -->
         <li>
-          <a href="http://news.qooxdoo.org">blog</a>
```

Comment:
the blog, btw will soon exist on news.qooxdoo.org again ... medium is setting this up for us